tpl/tplimpl/embedded/templates/shortcodes/youtube.html

```diff
@@ -4,7 +4,19 @@
 {{- $id := .Get "id" | default (.Get 0) -}}
 {{- $class := .Get "class" | default (.Get 1) -}}
 {{- $title := .Get "title" | default "YouTube Video" }}
+{{- $start := .Get "start" | default "" -}}
+{{- $autoplay := .Get "autoplay" | default "" -}}
+{{- $queryComponents := slice -}}
+{{- if $start -}}
+  {{- $queryComponents = $queryComponents | append "start" $start -}}
+{{- end -}}
+{{- if $autoplay -}}
+  {{- $queryComponents = $queryComponents | append "autoplay" 1 -}}
+{{- end -}}
+{{- $queryString := (querify $queryComponents | safeURL) | default "" -}}
+
 <div {{ with $class }}class="{{ . }}"{{ else }}style="position: relative; padding-bottom: 56.25%; height: 0; overflow: hidden;"{{ end }}>
-  <iframe src="https://{{ $ytHost }}/embed/{{ $id }}{{ with .Get "autoplay" }}{{ if eq . "true" }}?autoplay=1{{ end }}{{ end }}" {{ if not $class }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }}allowfullscreen title="{{ $title }}"></iframe>
+  <iframe src="https://{{ $ytHost }}/embed/{{ $id }}{{with $queryString}}{{ printf "?%s" . }}{{ end }}" {{ if not $class }}style="position: absolute; top: 0; left: 0; width: 100%; height: 100%; border:0;" {{ end }}allowfullscreen title="{{ $title }}"></iframe>
 </div>
 {{ end -}}
+
```

Comment:
Same remark re newline as above. It may be possible that your editor auto-adds newlines at the end of file. I always turn that option off.

Reply:
Also, there are some test failures (see the GitHub action status)